static/app/views/replays/utils/mergeBreadcrumbsEntries.tsx

```diff
@@ -4,11 +4,34 @@ function isBreadcrumbs(entry: Entry) {
   return entry.type === EntryType.BREADCRUMBS;
 }
 
-export default function mergeBreadcrumbsEntries(events: Event[]): Entry {
+function eventBreadcrumbEntries(event: Event) {
+  return event.entries.filter(isBreadcrumbs);
+}
+function entryValues(entry: Entry) {
```

Comment:
These functions aren't being reused, should we just inline these into the `flatMap` calls?

Reply:
I started with them inlined, and was writing comments to try to make it obvious what's happening and realized I was fundamentally writing these method names. So I pulled them up and out in order to make the `events.flatMap().flatMap().filter()` business more readable.

Context: This is part of the Replay loading code that will change in summer months once we get our backend up and running. TBD but we should have some kind of different objects to fetch, and wouldn't munge to much data on the front-end like this anymore.

Reply:
Looked at those two floating methods again, and I noticed I could roll everything together and typescript would be happier in the process. Before the types were getting lost inside the flatMaps and then everything was smashed into a string with the JSON.stringify() call, so I hadn't noticed that `any` was inferred. Now it's better, and we can see in vscode that `RawCrumb[]` is the return type.

<img width="521" alt="Screen Shot 2022-04-18 at 10 49 45 AM" src="https://user-images.githubusercontent.com/187460/163851296-5dda60c0-5c08-4d42-a24d-2acc41503f43.png">